src/loaders/jobs.ts

```diff
@@ -24,8 +24,13 @@
 
 import cron from 'node-cron'
 import { tlsChecker } from '../jobs/tls-check'
+import { check_db_size } from '../jobs/check-database'
+import { getConfig } from '../components/config'
 
 export function jobsLoader() {
+  const { db_limit } = getConfig()
   // schedule TLS checker every day at 00:00
   cron.schedule('0 0 * * *', tlsChecker)
+  // schedule database size check
+  cron.schedule(db_limit.cron_schedule, check_db_size)
```

Comment:
Do we need to check if db_limit.cron_schedule is exists to run this command?

Reply:
done